azure-toolkit-libs/azure-toolkit-common-lib/src/main/java/com/microsoft/azure/toolkit/lib/common/messager/IAzureMessager.java

```diff
@@ -0,0 +1,32 @@
+package com.microsoft.azure.toolkit.lib.common.messager;
+
+import javax.annotation.Nonnull;
+
+public interface IAzureMessager {
+
+    default boolean confirm(@Nonnull String message, String... title) {
```

Comment:
beside boolean confirm, I think there might be some other input prompt like:
select subscription
select resource group
select runtime

Reply:
I don't think so. messager is only for **output** but `prompt` is for **input**.